requirements.in

```diff
@@ -0,0 +1,2 @@
+aiohttp
+sqlalchemy
```

Comment:
It is not needed at the moment as we don't use postgres yet.
However, thank you for heads-up, we plan to use postgres as soon as the other basics get sorted out.